src/app/modules/submission/submission.utils.ts

```diff
@@ -68,14 +64,10 @@ const encryptFormFieldModeFilter = <T extends FormField>(
 
 // Exported for testing.
 export const getFormFieldModeFilter = (
-  responseMode: FormResponseMode,
+  encrypted: boolean,
 ): (<T extends FormField>(responses: T[]) => T[]) => {
-  switch (responseMode) {
-    case FormResponseMode.Email:
-      return emailResponseModeFilter
-    case FormResponseMode.Encrypt:
-      return encryptFormFieldModeFilter
-  }
+  if (encrypted) return encryptFormFieldModeFilter
+  else return emailResponseModeFilter
```

Comment:
same comment as `getResponseModeFilter`

Reply:
Done in [4bf6286](https://github.com/opengovsg/FormSG/pull/6595/commits/4bf6286022cba46dd59163f21e396c98fb21fbf5)! Hope the new names are good?

Reply:
Sounds good!